pkg/platform/riscv64.go

```diff
@@ -15,6 +16,18 @@ func (p *RISCV64) GetArch() arch.Arch {
 func (p *RISCV64) GetPackages() []string {
 	packages := p.BasePlatform.FirmwarePackages
 
+	if p.UEFIVendor != "" {
+		packages = append(packages,
+			// XXX: this is needed to get a generic bootkernel,
+			// this should probably be part of any bootable img
+			// packagelist
+			"dracut-config-generic",
```

Comment:
Are we sure about this one?

Reply:
Sort of sure, here is my reasoning:
1. I got the hint from @ondrejbudai (thanks for this!). 
2. We include it in all other platforms too so I think this part is okay. I also think this is not the best place AFAICT this is not arch specific, we have it on all arches.
3. The description of `dnf info dracut-config-generic` says: `This package provides the configuration to turn off the host specific initramfs generation with dracut and generates a generic image by default`

But maybe I'm missing something or misunderstanding the question? (or is this about the comment that is not clear enough?)

Reply:
You're not missing anything the misunderstanding was on my side!